### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by krisbobo**
## Project: Popup window:
**The following changes were introduced:**
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -227,6 +136,43 @@ <h3 class="deskt">Let's Chat!</h3>
         </div>           
     </section>
 
+    <!-- The Modal -->
+    <div id="myModal" class="modal">
```
Comment (danielufeli) [inline index.html]: - [ ] Nice working on your modal popup, kindly add a background just like you have on the Figma Template
- [ ] Also kindly dynamically populate the information of the modal with the data in the project array.

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @krisbobo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js [hunk 2]
```diff
@@ -76,3 +76,97 @@ document.addEventListener('DOMContentLoaded', () => {
...
+        </div>
+    </div>
+  `;
+
+  worksContainer.appendChild(workCards);
+});
+
+// ******* Modal Popup*******
+
+const modal = document.getElementsByClassName('modal');
```
Comment (Ismailco) [inline js/main.js]: - [ ] As the previous reviewer mentioned you should dynamically populate the information of the modal with the data in the project array, kindly implement this so your popup uses the right information

Comment (Ismailco) [inline js/main.js]: - [ ] [OPTIONAL] This is a peer-programming project, kindly in the future make sure to pay attention to this requirement, all the students in the group should contribute to this PR

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @krisbobo,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — COMMENTED** [review]
Hi @krisbobo,

Nice work you have here!
I like a lot how you implemented the dynamically render of the work cards with the use of JavaScript :clap:

What I need to add here are the followings:
- as this project is a pair-programming activity, the first reviewer, @danielufeli, should have invalidated this project because only one student submitted to this branch;
- both reviewers, @danielufeli and @Ismailco, correctly noticed that in your project, the pop-up is built in the HTML file and with JavaScript, the user controls if the pop-up is displayed or not;

In order to pass this milestone, kindly, change the following in the project:
- as the work cards are displayed dynamically, kindly, do the same with the pop-up;
Imagine that you have different projects that you want to show off in your portfolio. If a user will want to see one of your projects and he will click on the "See Project" button, each time, a different project should be displayed, and not the same pop-up, as it happens now.
I understand that for now you display only mocks and for all work cards, the pop-up is the same, but still, we need to render on the pop-up different information from the project array, depending on what project the user wants to see.

I hope is a bit more clear for you now!

Keep up the good work! :rocket:

**Comment by krisbobo** [comment]
Hi, @nicupop729 I made the pop-up window to display dynamically but the contents don't show up. I'm not sure of the number of reviews I might have but if it is possible, could you take a look at what I have done before I submit for a review again?

**Comment by krisbobo** [comment]
Hi @nicupop729!
Everything works well now. I'll merge to resolve issue #14

**Pull request merged**